Apps/VisionOS/VisionOS/AppDelegate.swift

```diff
@@ -0,0 +1,39 @@
+import CioTracking
+import UIKit
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication
+            .LaunchOptionsKey: Any]? = nil
+    ) -> Bool {
+        /*
+
+         How do I get the values for the siteId and apiKy?
+         1- Login or Signup in CustomerIO platform: https://fly.customer.io/
+         2- On the top right section click the
+         Settings Icon >> Workspace Settings >> API and webhook credentials
+         You will see list of one or more workspaces and each has
+         its own pair of siteId and apiKey, these are the ones
+         to use here.
```

Comment:
I do greatly appreciate your efforts to help customers find the information that they need in this code! 

I am concerned of this documentation getting out-of-date. Instead, do we have a webpage in our CIO docs site that we could point to here?

Reply:
I had similar comment earlier about the fact that the proximity to the code should give it better chance of not being outdated; with that said, a link here to a full document webpage won't hurt. On it